components/centraldashboard/public/components/main-page.pug

```diff
@@ -32,10 +33,10 @@ app-drawer-layout.flex(narrow='{{narrowMode}}',
                 paper-tabs.bottom(selected='[[page]]', attr-for-selected='page',
                         hides, hidden$='[[hideTabs]]')
                     paper-tab(page='dashboard', link)
-                        a.link(tabindex='-1', href='/') Dashboard
+                        a.link(tabindex='-1', href$='[[_buildHref("/")]]') Dashboard
                     paper-tab(page='activity', link)
-                        a.link(tabindex='-1', href='/activity') Activity
-                aside#NamespaceSelector(hidden$='[[!equals(page, "activity")]]')
+                        a.link(tabindex='-1', href$='[[_buildHref("/activity")]]') Activity
```

Comment:
Won't these be static links?? All the `_buildHref`?

Reply:
I guess that depends. I was thinking that namespace would behave similar to how projectId behaves in the GCP console. It's propagated along with every link even if the page being displayed doesn't use it to change its behavior.